src/backend.rs

```diff
@@ -211,28 +282,22 @@ impl Backend {
             });
         }
 
+        let name = add.name.clone().unwrap();
```

Comment:
Is it always non-optional? Is safe to just uwrap?

Reply:
At this point it will always be non-optional. Unless we forget about it and reuse the method... I'll fix it.

Reply:
Fixed